rpcs3/Emu/SysCalls/Modules/sceNp.cpp

```diff
@@ -825,8 +826,15 @@ s32 sceNpManagerGetStatus(vm::ptr<u32> status)
 {
 	sceNp.Warning("sceNpManagerGetStatus(status=*0x%x)", status);
 
-	// TODO: Support different statuses
-	*status = SCE_NP_MANAGER_STATUS_OFFLINE;
+	// TODO: Support more statuses
```

Comment:
It isn't related. One of the reasons I added this is to gather intel on which games use what parts of the SceNp* online API. I will be making this to be a proper implementation in the near future, when I plan to start working on a PSN emulation.